lib/main_common.pm

```diff
@@ -479,6 +479,15 @@ sub load_docker_tests {
     }
 }
 
+sub load_system_role_tests {
```

Comment:
so far this applies only to openSUSE, right? I just wonder if we should be explicit in the name to prevent confusion about the SLE system roles

Reply:
I'm planning on restructuring the Tumbleweed and Leap (15.1+) control.xml's and installation routine to use the SLE style system_roles

So I chose a name that will suit the rationalisation that will come with that effort, otherwise I risk falling into the trap of making a pointless separation between openSUSE & SLE when the direction of travel is in the opposite direction.

Reply:
agreed, makes sense